styles.css

```diff
@@ -136,14 +478,195 @@ ul.navbar-items {
   }
 
   .prof {
-    color: white;
-    font-size: 25px;
+    font-family: 'Inter', sans-serif;
+    color: #fff5e1;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
   }
 
   .desc {
     color: #f4f5f7;
     width: 300px;
-    line-height: 1.5rem;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px !important;
```

Comment:
- **[OPTIONAL]** KIndly avoid to use **!important** in your styles for the best practices of HTML & CSS.